addons/l10n_in_edi_ewaybill/models/res_company.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    l10n_in_edi_ewaybill_username = fields.Char("E-Waybill (IN) Username", groups="base.group_system")
+    l10n_in_edi_ewaybill_password = fields.Char("E-Waybill (IN) Password", groups="base.group_system")
+    l10n_in_edi_ewaybill_auth_validity = fields.Datetime("E-Waybill (IN) Valid Until", groups="account.group_system")
```

Comment:
account.group_system?

Reply:
yes, because this is a change from Setting only.
The same already discuss in E-invoice and in last we decided to go with 'account.group_system'

Reply:
Ohh sorry, it's not account.group_system but it's base.group_system